Animated logo carousel

yathomasi opened the pull request:
Closes #4214 
- Preview Link: https://dvc-org-moving-company--r5kosr.herokuapp.com/
- inspirations from nextjs, vercel websites
- used the logos provided by @jendefig 
- logos are set to gray color by default and colorful on hover

omesser commented on src/components/Home/LogosSlider/index.tsx:
```diff
@@ -0,0 +1,39 @@
+      <div className="mb-5">
+        <h2 className={cn('text-2xl font-medium text-center', styles.title)}>
+          Empowering from startups to Fortune 500 Companies: Our Open Source
+          Tools at Work
```
Suggest this:
> Empowering thousands of users and customers from startups to fortune 500 companies

notes:
- avoiding open source mention - some use FOSS, some paid, and future rebranding might make this more misleading, so keeping it general
- sentence case (not title case)
- don't think we need ":" here (also avoided ",") - will look cleaner imo

jendefig replied:
@omesser 
>avoiding open source mention - some use FOSS, some paid, and future rebranding might make this more misleading, so keeping it general

Just for context: We have on purpose made the dvc.org website about open source (see hero). 
All of these logos will not be able to be used for Studio. They can only be used for DVC website (or eventual page in rebranding) except for the ones that came from the Iterative site. So the open-source part is not an issue here.

That being said, I like your edit.  Because open-source is mentioned in the hero, this simplifies the idea into one thought: EMPOWER, and lets people reflect on logos. I like it.

omesser replied:
> We have on purpose made the dvc.org website about open source (see hero).

Yep, that makes total sense @jendefig  - makes sense IMO to refer here to both users and customers, in the page-context of open source like you mentioned!

omesser reviewed (changes requested):
Looks great @yathomasi !! ❤️ 
Suggested some rephrasing for the sentence, PTAL

jendefig commented:
OMIGOD!!!!  LOOOOOOOVE!!!! 🙌🏼

Couple things:
Can we move the sentence under the logos. or in some way tighten it up because on my screen I can't see the logos when I arrive: 
<img width="1691" alt="image" src="https://user-images.githubusercontent.com/34289199/230625118-24626bce-2436-4303-a66e-9add44fc2de6.png">


I'm noticing that if you put the sentence under the logos, it will look awkward with the next section.  (which looks a bit awkward now because white on white sections) We may need to do something about the background in the next section. 

Possible easy fix that might be helpful in the meantime until we figure out the rest: Duplicate the bottom block to join the newsletter after the logos and then continue with the rest of the page. Might be nice to test if this makes an impact while we are figuring out next steps.  It has little function at the bottom of the page.  Most of our new newsletter emails are coming from the course.

jendefig reviewed (changes requested):
Love it!  See the other comment I put separately in my excitement! 👆🏽

yathomasi commented:
Thanks @omesser  and @jendefig  for the suggestions.

> tighten it up because on my screen I can't see the logos when I arrive

moved up to hero section so it should be visible now

> it will look awkward with the next section

Again, now we moved to hero section the coloring looks good to me

> join the newsletter after the logos and then continue with the rest of the page.

Sure, we can test that. I will push an update soon.

> Can we increase the speed a bit?

I think it's more faster than previous and few other optimizations 

> Can you move UBS further up in the lineup - Let's put it after InLAB.

Done, right after InLAB. Let me know if we have more similar preferences. 


> Then I noticed that once they go through there is a flicker and they jump back after one cycle. Can you double the set of logos?

As mentioned, few optimizations are in place so that the flicker seems invisible or slight. We already had double set, now I multiplied it few more times and it should be negligible now.

> Empowering thousands of users and customers from startups to fortune 500 companies

Updated title with
1. `fortune` -> `Fortune`
2. Also, added `:` to end similar to (https://iterative.ai/). Let me know if we have strong opinion on this.

omesser commented:
> fortune -> Fortune
Also, added : to end similar to (https://iterative.ai/). Let me know if we have strong opinion on this.

I personally think it looks a bit jarring and is slicker without the capital "F" and the ":". But I don't feel strongly about this that much 😄 , definitely not blocking
Will let you and @jendefig decide.
I think there's some restyler errors still - I'm approving to unblock  - great job @yathomasi 🙌

omesser commented:
@yathomasi  - can you TAL at the restyled error please? is it a false negative?

yathomasi commented:
> @yathomasi - can you TAL at the restyled error please? is it a false negative?

Kind of, it's not actionable on our side. I had an issue open on their side https://github.com/restyled-io/restyler/issues/204 and as mentioned the solution is not applicable or worth the effort. So, I believe it's okay to skip that.

jendefig reviewed (approved):
Love it! 
Regarding Fortune 500 - It should be capitalized. It came from the magazine Fortune, so capitalized is correct.

The only thing is that there a way that the logos can fade out at the actual sides of my screen as opposed to the abbreviated one in line with the copy? 
<img width="1743" alt="image" src="https://user-images.githubusercontent.com/34289199/231212794-956b1cd0-ac2a-497f-bd86-06e750a73fe4.png">

jendefig commented:
> so that it doesn't look overflowing.

We want the logos to look overflowing.  Before the changes they went completely across my screen. See my previous picture.

yathomasi commented:
> We want the logos to look overflowing. Before the changes they went completely across my screen. See my previous picture.

<img width="2056" alt="Screenshot 2023-04-12 at 09 12 16" src="https://user-images.githubusercontent.com/20840228/231341125-2cabefc6-d5df-44ad-85f3-6f7619edcd06.png">

This is how it would look and we wouldn't recommend it for a few good reasons:
- we don't have content in the whole page overflowing outside the current container limit and we would want to be consistent
- The main reason we don't want to hit the screen is it will not have great UX for large screens, wide screen monitors
<img width="1658" alt="Screenshot 2023-04-12 at 09 19 21" src="https://user-images.githubusercontent.com/20840228/231342248-a66a8cc7-e41a-41b7-914f-f72d66a4b780.png">

- Also, I don't think we want to overhaul with lots of logos

yathomasi commented:
Merging this as we don't have any major blocker. We can definitely do follow-ups if anything comes up. cc: @jendefig

dberenbaum commented:
Great work @yathomasi @jendefig! Looks nice.
